ls1intum/Artemis
# [Chore] - Update to jhipster 7.1.0

### Checklist

### Motivation and Context


A major update to jhipster 7.1.0 has to be done.

### Description

We updated to jhipster 7.1.0 and adapted the code accordingly. A major change is the alert component, which is not imported by jhipster anymore but rather implemented on our own.

### Steps for Testing


1. Log in to Artemis
2. Try out different Artemis features like creating lectures, conducting exercises and exams
3. Trigger alerts and make sure those are shown correctly
4. Make sure translations work correctly

### Review Progress





- Code Review
- Manual Tests
